Documentation/README.md

```diff
@@ -3,6 +3,8 @@ Quick Links for CEP
 The page provides links to [get started](https://github.com/Adobe-CEP/Getting-Started-guides) with Common Extensibility Platform (CEP) in order to create extensions for Adobe Creative Cloud applications. 
 
 **HTML Extension Cookbook**
+* [CEP 12.0](./../CEP_12.x/Documentation/CEP%2012%20HTML%20Extension%20Cookbook.md)
```

Comment:
Can you update the link for previous versions as well. It is good to have links pointing to same git branch.

Reply:
Updated